ckanext/unaids/theme/templates/user/edit_user_form.html

```diff
@@ -6,17 +6,28 @@
   <fieldset>
     <legend>{{ _('Change details') }}</legend>
 
-    <p>{{ _('To change your profile settings, please ') }}<a href="{{ h.get_ape_url() }}">{{ _('click here') }}</a>.</p>
+    <p style="font-weight: bold; margin-bottom: 0.4em">{{ _('Profile data') }}:</p>
```

Comment:
NEVER use in-line CSS styling, unless you have a very specific reason for doing so. This is because it can't easily be overridden by the CSS cascade. 

Use the developer console to inspect the other form labels.  If you do this, you should realise that instead of using `<p></p>` you should use `<label></label>`. This should format the label in the same manner as all the other form labels around the site.